scooby/tracker.py

```diff
@@ -0,0 +1,79 @@
+from scooby.report import Report
+from scooby.knowledge import get_standard_lib_modules
+
+TRACKING_SUPPORTED = False
+SUPPORT_MESSAGE = "Tracking is not supported for this version of Python. Try using a modern version of Python."
+try:
+    import builtins
+    CLASSIC_IMPORT = builtins.__import__
+    TRACKING_SUPPORTED = True
+except (ImportError, AttributeError):
+    pass
+
+# The variable we track all imports in
+TRACKED_IMPORTS = ["scooby"]
+
+MODULES_TO_IGNORE = {
+    "pyMKL",
+    "mkl",
+    "numexpr",
```

Comment:
Why do you ignore `numexpr`? In my package `empymod` you can use or not use `numexpr`, which has quite some difference (in speed). So I would want to now if numexpr is there/was used or not.

Ignoring `mkl` is fine, as it is reported if available anyway.

Reply:
I ignored it here because it was used here: 

https://github.com/banesullivan/scooby/blob/72d1af296bd12ff016ad3f1dccaef94476c3fd96/scooby/knowledge.py#L42

If you think it should be included, then I can delete that!

Reply:
I see. Difficult one. So scooby will always load it if possible... Are there other packages like this?

Anyway, I think we should take it out. If a project uses it, but it then doesn't show up in the list, it will cause confusion...